base/parse.jl

```diff
@@ -167,6 +167,32 @@ float(x::AbstractString) = parse(Float64,x)
 
 float{S<:AbstractString}(a::AbstractArray{S}) = map!(float, similar(a,typeof(float(0))), a)
 
+function parse{T<:AbstractFloat}(::Type{T}, s::AbstractString, base::Integer)
+    s = strip(s, ' ')
```

Comment:
These Benchmarks are not to be trusted - im running them on a CoreM CPU which throttles after a short boost period. I think, the second Benchmark had some throttled CPU-time.